pkg/apiserver/apiserver_test.go

```diff
@@ -2699,3 +2700,23 @@ func TestUpdateChecksAPIVersion(t *testing.T) {
 		t.Errorf("unexpected response: %s", string(b))
 	}
 }
+
+func TestEncodeStatus(t *testing.T) {
```

Comment:
Is apiserver_test.go the best place for this test? Maybe move it into pkg/conversion somewhere?